src/item_location.cpp

```diff
@@ -22,7 +22,7 @@ class item_location::impl
     public:
         virtual ~impl() = default;
         virtual std::string describe( const Character * ) const = 0;
-        virtual int obtain( Character & ) = 0;
+        virtual int obtain( Character &ch, int qty ) = 0;
```

Comment:
This was done for the charge cards implementation (b0ff445a32f140092b2a01b322fb20ade9906d6a, maybe the author @i2amroy can tell us more). I don't know whether this is actually important. On my platform `int` has the same size as `long`, so it doesn't matter.

But it is much more consistent.